### DISCUSSION TITLE
KeyNotFound - CodingKeys(stringValue: "accessList") Swift. Web3Wallet-sdk

### DISCUSSION THREAD

**Discussion by 2jumper3** [WalletConnect/walletconnect-monorepo#4039] [Developer Support]
ChatGPT
Hello everyone!
I'm in the process of integrating the web3wallet and encountered a strange issue. I'm using [your resource](https://react-app.walletconnect.com/) to check the functionality of the application, and when calling the eth_sendTransaction method as outlined in the documentation, the app crashes because it expects data that is not present in the request. For example, the EthereumTransaction structure has a mandatory field called accessList (see screenshot 1). 
<img width="660" alt="Screenshot 2023-12-12 at 15 41 48" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/45697468/26809736-476f-4757-9b66-f8b8ccc91b3a">
However, it is not included in the request. Are there any suggested solutions? Currently, the implementation below is taken from your example located in the path shown in the screenshot below.
<img width="704" alt="Screenshot 2023-12-12 at 15 54 06" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/45697468/7db000ea-6afa-4035-9003-ac7fc1563715">

<img width="298" alt="Screenshot 2023-12-12 at 15 43 39" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/45697468/ee45dca2-90b2-438e-8d2f-584059284985">

And the last thing, do you have any examples with Tron?) Thanks a lot!)

**Comment by glitch-txs** [comment]
hi, could you open an issue in the swift repo for this? Thanks!
